configure.ac

```diff
@@ -361,15 +361,25 @@ AC_SUBST([GIRTYPELIBDIR])
 AM_CONDITIONAL([ENABLE_INTROSPECTION],[test "$enable_introspection" = "yes"])
 
 dnl ================== libsynctex ===========================================
+AC_ARG_ENABLE([synctex],
+  [AS_HELP_STRING([--disable-synctex], [Disable support for synctex])],
+  [],
+  [enable_synctex=yes])
```

Comment:
only two values are allowed: "yes", "no"
```diff
--- a/configure.ac
+++ b/configure.ac
@@ -363,9 +363,11 @@ AM_CONDITIONAL([ENABLE_INTROSPECTION],[test "$enable_introspection" = "yes"])
 dnl ================== libsynctex ===========================================
 AC_ARG_ENABLE([synctex],
   [AS_HELP_STRING([--disable-synctex], [Disable support for synctex])],
-  [],
-  [enable_synctex=yes])
-
+  [case "${enableval}" in
+    yes) debug=true ;;
+    no)  debug=false ;;
+    *) AC_MSG_ERROR([bad value ${enableval} for --disable-synctex or --enable-synctex]) ;;
+  esac],[enable_synctex=yes])
 
 if test "$enable_synctex" = "yes"; then
   AC_DEFINE([ENABLE_SYNCTEX],[1],[Define if synctex support is enabled])
```
Test before:
```
$ ./autogen.sh --prefix=/usr --enable-synctex=ASDF
<cut>
    Synctex enabled.....:    ASDF
<cut>
```
Test after:
```
$ ./autogen.sh --prefix=/usr --enable-synctex=ASDF
<cut>
configure: error: bad value ASDF for --disable-synctex or --enable-synctex
```

Reply:
I implemented your suggested patch in a slightly different way, but the meaning is the same.